### PR TITLE
GUI demos

### PR DESCRIPTION
**Pull request by LB0ge**
Asana task link: https://app.asana.com/0/0/1204724384298610/f

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

**Comment by sebastianbochan** [comment]
One remark, should it be `2 x nested` instead of `1 x nested`?

Screenshot 2023-06-22 at 11 56 19

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: samples/dashboards/gui/dimensions/demo.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<div id="container"></div>
+
+<pre id="csv">
+Food,Vitamin A
+Beef Liver,6421
+Lamb Liver, 2122
+Cod Liver Oil, 1350
+Mackerel, 388
+Tuna, 214</pre
+>
```
Comment (pawellysy) [inline samples/dashboards/gui/dimensions/demo.html:15]:
```suggestion
Tuna, 214</pre>
```

**Review by pawellysy — APPROVED** [review]
Great job! Just one typo fix

File: samples/dashboards/gui/dimensions/demo.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        {
+                            cells: [
+                                {
+                                    id: 'dashboard-col-0',
+                                    width: '60%',
+                                    height: 350
+                                },
+                                {
+                                    id: 'dashboard-col-1',
+                                    height: 450
```
[the remark this answers is not in the document]
Comment (LB0ge) [reply samples/dashboards/gui/dimensions/demo.js:32]: `width` is already declared as string (%) in line 27. Should I declare one `height` as a string too?

[the remark this answers is not in the document]
Comment (karolkolodziej) [reply samples/dashboards/gui/dimensions/demo.js:32]: You are right, I missed that- no there is no need to do so ;)

**Review by karolkolodziej — APPROVED** [review]

**Comment by LB0ge** [comment]
> One remark, should it be `2 x nested` instead of `1 x nested`?
> 
> Screenshot 2023-06-22 at 11 56 19

I believe this is correct as is. All three cells marked `2 x nested` are inside a layout which is inside another layout (ref. board configuration).

**Comment by sebastianbochan** [comment]
> > One remark, should it be `2 x nested` instead of `1 x nested`?
> > Screenshot 2023-06-22 at 11 56 19
> 
> I believe this is correct as is. All three cells marked `2 x nested` are inside a layout which is inside another layout (ref. board configuration).

Thank you for elaboration, now makes sense. 
Agree ;)

File: samples/dashboards/gui/dimensions/demo.html (hunk 1 above)
Comment (LB0ge) [reply samples/dashboards/gui/dimensions/demo.html:15]: I will fix!

**Review by sebastianbochan — APPROVED** [review]

**Comment by sebastianbochan** [comment]
LGTM 👍

**Review by pawellysy — APPROVED** [review]
LGTM

**Review by karolkolodziej — APPROVED** [review]

**Review by bre1470 — APPROVED** [review]
LGTM

**Pull request merged**